src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportControlCheckTest.java

```diff
@@ -439,6 +442,29 @@ public void testBeginTreeCurrentImportControl() throws Exception {
         verifyWithInlineConfigParser(file1, file2, expectedFirstInput, expectedSecondInput);
     }
 
+    @Test
+    public void testImportControlFileName() throws Exception {
+        final String[] expected = {
+            "11:1: " + getCheckMessage(MSG_DISALLOWED, "java.awt.Image"),
+        };
+
+        verifyWithInlineConfigParser(
+                getPath("InputImportControlTestRegexpInFile.java"), expected);
+    }
+
+    @Test
+    public void testImportControlTestException() throws IOException {
+        final String filePathOfImportControlXml = getPath("InputImportControlTestException.xml");
+        final CheckstyleException ex = assertThrows(CheckstyleException.class, () -> {
+            verifyWithInlineConfigParser(getPath("InputImportControlTestException.java"));
+        });
+
+        assertThat(ex.getCause().getCause().getCause().getCause().getCause().getMessage())
+                .isEqualTo("unable to parse file:" + filePathOfImportControlXml
```

Comment:
in Windows message duet path is different, like `unable to parse file:C:\projects\checkstyle\`
to avoid this problem you need to split messages validation in two statements
`.startsWith("unable to parse file:")` and `.endsWith(" - Documen root ........")`

Reply:
Done.